### PR TITLE
separate anotation processors into separate modules

### PR DESCRIPTION
**Pull request by zdenek-jonas**

### CODE REVIEW

File: codegen/entity/src/main/java/module-info.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License, v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is
+ * available at https://www.gnu.org/software/classpath/license.html.
+ * 
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ * #L%
+ */
+module microstream.entity.codegen
```
Comment (fh-ms) [inline codegen/entity/src/main/java/module-info.java]: naming convention: -> microstream.codegen.entity

File: codegen/wrapping/src/main/java/module-info.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License, v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is
+ * available at https://www.gnu.org/software/classpath/license.html.
+ * 
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ * #L%
+ */
+module microstream.wrapping.codegen
```
Comment (fh-ms) [inline codegen/wrapping/src/main/java/module-info.java]: naming convention: -> microstream.codegen.wrapping

**Review by fh-ms — CHANGES_REQUESTED** [review]
Package names should follow convention as well:
one.microstream.entity.codegen -> one.microstream.codegen.entity
one.microstream.wrapping.codegen -> one.microstream.codegen.wrapping

File: codegen/entity/src/main/java/module-info.java (hunk 1 above)
Comment (zdenek-jonas) [reply codegen/entity/src/main/java/module-info.java]: Updated

File: codegen/wrapping/src/main/java/module-info.java (hunk 2 above)
Comment (zdenek-jonas) [reply codegen/wrapping/src/main/java/module-info.java]: updated

**Comment by zdenek-jonas** [comment]
> Package names should follow convention as well: one.microstream.entity.codegen -> one.microstream.codegen.entity one.microstream.wrapping.codegen -> one.microstream.codegen.wrapping

I would prefer to keep it. It could be a breaking change.

**Review by fh-ms — APPROVED** [review]

**Pull request merged**